A few minor fixes & features

desht opened the pull request:
Nothing major

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/FTBTeam/FTB-Library-Legacy?pullRequest=86) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/FTBTeam/FTB-Library-Legacy?pullRequest=86) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/FTBTeam/FTB-Library-Legacy?pullRequest=86) it.</sub>

desht committed:
chore: fixed up PanelPositioning

Also add SNBTConfig#addEnum variant to allow custom default to be specified

desht committed:
fix: fix color selector hue/sat wandering when brightness adjusted

desht committed:
build: version -> 2004.2.2

desht committed:
chore: better gui size calc for EditConfigScreen

desht committed:
chore: use TextField widget for drawing EditConfigScreen title

Better if the title is very long (won't overflow gui bounds due to default panel
scissoring behaviour)

MichaelHillcox commented:
Closing due to archived mod
